src/genie/libs/parser/iosxe/tests/ShowWirelessStatsMobility/cli/equal/golden_output_output.txt

```diff
@@ -0,0 +1,95 @@
+Mobility event statistics:
+    Joined as
+        Local                         : 60431           
+        Foreign                       : 0               
+        Export foreign                : 0               
+        Export anchor                 : 0               
+    Delete
+        Local                         : 60316           
+        Remote                        : 0               
+    Role changes
+        Local to anchor               : 0               
+        Anchor to local               : 0               
+    Roam stats
+        L2 roam count                 : 0               
+        L3 roam count                 : 0               
+        Flex client roam count        : 0               
+        Inter-WNCd roam count         : 0               
+        Intra-WNCd roam count         : 154227          
+        Remote inter-cntrl roam count : 0               
+        Remote WebAuth pending roams  : 0               
+    Anchor Request
+        Sent                          : 0               
```

Comment:
Please let me know how we should handle these lines. They have numbers associated, but the indenting would suggest that the values below should be part of a `"sent"` and `"received"` dict

Reply:
Hi, is it a total count for ```Grant received``` and ```Deny received```? If yes, you can add a total key under sent dict
```
 {
    "anchor_request": {
        "sent": {
            "grant_received": 0,
            "deny_received": 0,
            "total": 0,
        }
        "received": {
            "grant_sent": 0,
            "deny_sent": 0,
            "total": 0,
        },
    }
}

Reply:
It looks like a separate count, not a total. Might be better to leave as is.
<img width="953" alt="image" src="https://user-images.githubusercontent.com/13859438/97014564-7d44ee80-1518-11eb-824f-c664c04f6858.png">
This is the only resource I could find on the topic: 
http://what-when-how.com/deploying-and-troubleshooting-cisco-wireless-lan-controllers/troubleshooting-mobility-cisco-wireless-lan-controllers-part-1/

Reply:
Ok, sounds good! Thank you for the information.